inductiva/molecules/scenarios/_post_processing.py

```diff
@@ -0,0 +1,100 @@
+"""Post process Gromacs simulation outputs."""
+from typing import Optional
+
+from absl import logging
+
+import os
+import MDAnalysis as mda
+import nglview as nv
+
+from uuid import UUID
+
+from inductiva.types import Path
+from inductiva.molecules.simulators import GROMACS
+from inductiva.utils.templates import (TEMPLATES_PATH,
+                                       replace_params_in_template)
+from inductiva.tasks import fetch_task_output, get_task_info
+
+import json
+
+SCENARIO_TEMPLATE_DIR = os.path.join(TEMPLATES_PATH, "protein_visualization")
+GROMACS_TEMPLATE_INPUT_DIR = "gromacs"
+
+
+class GROMACSSimulationOutput:
+    """Post process Gromacs simulation outputs."""
+
+    def __init__(self, task_id: str, sim_output_path: Path = None):
+        """Initializes a `SimulationOutput` object.
+
+        Currently only supports asynchronous simulations."""
+
+        if not get_task_info(task_id)["status"]:
+            raise ValueError("Simulation not finished.")
+
+        self.sim_output_dir = sim_output_path
+
+        if sim_output_path is None:
+            self.sim_output_dir = os.path.join(os.getcwd(),
+                                               f"inductiva_output/{task_id}")
+
+        fetch_task_output(task_id, output_dir=sim_output_path, return_type=None)
```

Comment:
I think we should avoid this pattern where a visualization class/function is also performing other tasks such as downloading task outputs. That should be done separately. All the visualization helper should do is take files stored locally and produce image/video(s). I'd only accept `sim_output_path` as an argument to `__init__`.

Reply:
However, this already poses a nice methodology to run things remotely. So in the future this might be required. But I agree with Fabio, that for now this should not occur, and this post-processing could happen on the scenario side.